Resolves 526, create form element component

ClassicSours opened the pull request.

tabersays commented on src/app/app.module.ts:
```diff
@@ -13,11 +13,13 @@ import {ConfigService} from './config.service';
   { path: 'sandbox', component: SandboxComponent },
-  { path: 'contributing', component: ContributingComponent }
+  { path: 'contributing', component: ContributingComponent },
+  { path: 'element', component: BossyFormElementComponent }
```
This additional path likely should not be in the PR

tabersays commented on src/bossy-ui/components/form-element/form-element.css:
```diff
@@ -0,0 +1 @@
+
```
Useless file remove

tabersays commented on src/bossy-ui/components/form-element/form-element.enum.ts:
```diff
@@ -0,0 +1,13 @@
+export enum BossyFormElementEnum {
+  calendar,
+  dropdown,
+  // dropdownmenuitem,
```
These comment pose no use and should be removed

glawson replied:
Don't call it an enum - it's not.  Please add as a 'string constant' in the FormConstantService - #529

glawson commented:
@ClassicSours - please update, see Taber's comments.

ClassicSours commented:
Sorry, updated with https://github.com/buildcom/BossyUI/tree/form-element
Uses fork and pull method instead of pull request
removed unnescesary file (enum that did nothing) and added css to form-element.css
resolves ng lint issue and passes codeship
included config's for each element
